linux/Platform.c

```
@@ -991,6 +993,24 @@ void Platform_getBattery(double* percent, ACPresence* isOnAC) {
    Platform_Battery_cacheTime = now;
 }
 
+int Platform_getThinkpadFan(void) {
```

Cppcheck (2.20.0)

[style] The function 'Platform_getThinkpadFan' is never used.

(unusedFunction)